SKRecording/VRHand.cs

```diff
@@ -28,56 +30,89 @@ public JointInfo(FingerId fingerId, JointId jointId, ModelNode fingerNode)
+
+            //adding the rig bones to the ointinfo class
+            foreach (JointInfo j in jointInfo)
             {
-                handModel = Model.FromFile("rightHand.glb");
+                if (j.rootBone)
+                {
+                    string jointName = "Hand.Wrist." + whichHand.ToString();
+                    j.node = handModel.FindNode(jointName);
+
+                }
+                else
+                {
+                    string jointName = j.finger.ToString() + "." + j.joint.ToString() + "." + whichHand.ToString();
+                    j.node = handModel.FindNode(jointName);
+                }
+
             }
-            jointInfo = new JointInfo[] {
-                new JointInfo(FingerId.Thumb, JointId.KnuckleMajor, handModel.FindNode("ThumbMeta")),
-                new JointInfo(FingerId.Thumb, JointId.KnuckleMid,   handModel.FindNode("ThumbProxi")),
-                new JointInfo(FingerId.Thumb, JointId.KnuckleMinor, handModel.FindNode("ThumbDist")),
-
-                new JointInfo(FingerId.Index, JointId.Root,         handModel.FindNode("IndexMeta")),
-                new JointInfo(FingerId.Index, JointId.KnuckleMajor, handModel.FindNode("IndexProxi")),
-                new JointInfo(FingerId.Index, JointId.KnuckleMid,   handModel.FindNode("IndexInter")),
-                new JointInfo(FingerId.Index, JointId.KnuckleMinor, handModel.FindNode("IndexDist")),
-
-                new JointInfo(FingerId.Middle, JointId.Root,         handModel.FindNode("MiddleMeta")),
-                new JointInfo(FingerId.Middle, JointId.KnuckleMajor, handModel.FindNode("MiddleProxi")),
-                new JointInfo(FingerId.Middle, JointId.KnuckleMid,   handModel.FindNode("MiddleInter")),
-                new JointInfo(FingerId.Middle, JointId.KnuckleMinor, handModel.FindNode("MiddleDist")),
-
-                new JointInfo(FingerId.Ring, JointId.Root,         handModel.FindNode("RingMeta")),
-                new JointInfo(FingerId.Ring, JointId.KnuckleMajor, handModel.FindNode("RingProxi")),
-                new JointInfo(FingerId.Ring, JointId.KnuckleMid,   handModel.FindNode("RingInter")),
-                new JointInfo(FingerId.Ring, JointId.KnuckleMinor, handModel.FindNode("RingDist")),
-
-                new JointInfo(FingerId.Little, JointId.Root,         handModel.FindNode("PinkyMeta")),
-                new JointInfo(FingerId.Little, JointId.KnuckleMajor, handModel.FindNode("PinkyProxi")),
-                new JointInfo(FingerId.Little, JointId.KnuckleMid,   handModel.FindNode("PinkyInter")),
-                new JointInfo(FingerId.Little, JointId.KnuckleMinor, handModel.FindNode("PinkyDist"))};
-
-            defaultBoneRot = Quat.FromAngles(-90f, 0, 0);
-            nodeScale = 1;
         }
 
         // Display a hand with the provided joint information
         public void show(RecordingData[] data)
         {
-            if (data.Length != 25)
+            if (data.Length != 26)
```

Comment:
Didn't expect that this value might change when I initially hardcoded it in. Might be smarter to set a constant somewhere (e.g. as a static variable of the VRHand class) in the case that this number changes again

Reply:
added and integrated
```
static class Constants
    {
        // Server we connect to 
        public const string IP = "192.168.1.159";
        public const int port = 12345;
        public const int Count = 26; // how many joints per hand
    }
```